watertap/unit_models/zero_order/tests/test_photothermal_membrane_zo.py

```diff
@@ -0,0 +1,181 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+Tests for zero-order photothermal membrane model
+"""
+import pytest
+
+from io import StringIO
+
+from pyomo.environ import Block, Var, Constraint, ConcreteModel, value, assert_optimal_termination
+from pyomo.util.check_units import assert_units_consistent
+
+from idaes.core import FlowsheetBlock
+from idaes.core.util import get_solver
+from idaes.core.util.model_statistics import degrees_of_freedom
+from idaes.core.util.testing import initialization_tester
+from idaes.generic_models.costing import UnitModelCostingBlock
+
+from watertap.unit_models.zero_order import PhotothermalMembraneZO
+from watertap.core.wt_database import Database
+from watertap.core.zero_order_properties import WaterParameterBlock
+from watertap.core.zero_order_costing import ZeroOrderCosting
+
+solver = get_solver()
+
+class TestPhotothermalMembraneZO:
+    @pytest.fixture(scope="class")
+    def model(self):
+        m = ConcreteModel()
+        m.db = Database()
+
+        m.fs = FlowsheetBlock(default={"dynamic": False})
+        m.fs.params = WaterParameterBlock(
+                default={"solute_list": ["nitrogen"]})
+
+        m.fs.unit = PhotothermalMembraneZO(default={
+                        "property_package": m.fs.params,
+                        "database": m.db})
+
+        m.fs.unit.inlet.flow_mass_comp[0, "H2O"].fix(120)
+        m.fs.unit.inlet.flow_mass_comp[0, "nitrogen"].fix(1)
+
+        return m
+
+    @pytest.mark.unit
+    def test_build(self, model):
+        assert model.fs.unit.config.database is model.db
+        assert model.fs.unit._tech_type == "photothermal_membrane"
+
+    @pytest.mark.component
+    def test_load_parameters(self, model):
+        data = model.db.get_unit_operation_parameters("photothermal_membrane")
+
+        model.fs.unit.load_parameters_from_database(use_default_removal=True)
+        assert model.fs.unit.recovery_frac_mass_H2O[0].fixed
+        assert model.fs.unit.recovery_frac_mass_H2O[0].value == \
+               data["recovery_frac_mass_H2O"]["value"]
+
+        for (t, j), v in model.fs.unit.removal_frac_mass_solute.items():
+            assert v.fixed
+            if j in data["removal_frac_mass_solute"]:
```

Comment:
Should this be `if j **not** in data["removal_frac_mass_solute"]: `? Although the test will pass because the removal fractions are all 0 for the components you added to the yaml, if one were to update the yaml for any reason and include non-zero values, the test would fail. Moreover, this isn't checking what we're intending to check here.

Reply:
You're right, I got it backwards here. Good catch, I just pushed your suggested fix.